bitnami/minio/README.md

```diff
@@ -265,6 +266,13 @@ As an alternative, this chart supports using an initContainer to change the owne
 
 You can enable this initContainer by setting `volumePermissions.enabled` to `true`.
 
+### Adjust persistent volume access mode for standalone mode with multiply replicas
+By default, the standalone mode is running with a single replica `deployment.replicaCount=1` and you shall use the distributed mode in most of the use cases to achieve a multi-pod solution.
+
+If you like to have standalone with multi replicas than you have 2 options:
```

Comment:
than -> then

I would also specify in which case you may want to use the standalone with multi-replicas instead of the distributed